.github/workflows/next-release.yml

```
@@ -30,7 +30,7 @@ jobs:
       - name: Installation && Build SDK
         run: yarn --frozen-lockfile && yarn bootstrap-ci
       - name: Lint
-        run: yarn lerna-workspaces run lint
+        run: yarn lerna-workspaces run lint && cd docusaurus && npx prettier --write '**/*.mdx
```

```suggestion
        run: yarn lerna-workspaces run lint && npx prettier --check 'docusaurus/**/*.mdx
```
Wouldn't it make more sense for the CI to check if it's formatted rather than formatting the files?

---

I agree here that we can have a check command.